packages/app/components/claim/claim-button.tsx

```diff
@@ -43,8 +43,7 @@ export enum ClaimStatus {
 export const getClaimStatus = (edition?: CreatorEditionResponse) => {
   if (!edition) return undefined;
   if (
-    edition.total_claimed_count ===
-    edition.creator_airdrop_edition?.edition_size
+    edition.total_claimed_count >= edition.creator_airdrop_edition?.edition_size
```

Comment:
We need to add a condition that checks if the `edition size of edition.creator_airdrop_edition` is not equal to 0.